nextjs-13/src/app/auth/sign-in/page.test.tsx

```diff
@@ -0,0 +1,44 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { redirect } from 'next/navigation';
+import { useSession, signIn } from 'next-auth/react';
+
+import SignInPage from './page';
+
+jest.mock('next-auth/react');
+jest.mock('next/navigation');
+
+describe('SignInPage', () => {
+  describe('Session status is "loading', () => {
+    it('renders loading state', () => {
+      useSession.mockReturnValue({ status: 'loading' });
+      render(<SignInPage />);
+      expect(screen.getByText('Loading...')).toBeInTheDocument();
+    });
+  });
```

Comment:
Let's add a line break between each example for readability. Make the code breath 🌬️

Reply:
Done! The commit: https://github.com/nimblehq/js-framework-benchmark/pull/94/commits/ba8753a500152042f44be57e24e41df04e0e460c